test/test_ranker.py

```diff
@@ -67,30 +67,63 @@ def test_term_query_weight():
 
     eq_(round(term_weight,4), round(weight,4))
 
+@with_setup(create_dictionary_index, remove_test_dict_info)
+def test_conjunctive_score_calc():
+    query_terms = ['term', 'to', 'evaluate']
+
+    scores = {'15da4df3-9ef1-4e1a-b0ba-f93bf05a25d0': 0.9998169851514684, 
```

Comment:
Is difficult to actually know the exact value. Conjunctive search is correct name for AND search.